image/discourse_bench/Dockerfile

```diff
@@ -17,11 +19,12 @@ RUN gem install facter &&\
 
 # reconfigure PostgreSQL template databases to use UTF-8
 # otherwise rake db:create fails
-RUN pg_dropcluster --stop 9.3 main &&\
-    pg_createcluster --start -e UTF-8 9.3 main
+RUN pg_dropcluster --stop 9.5 main &&\
```

Comment:
I've done both of these:

```
docker run --rm discourse/discourse_test:1.3.7
docker run --rm discourse/discourse_bench:1.3.7
```

The tests all passed, and bench got through all three benchmarks (both as admin and not), but then threw an EACCES exception (could not open `/root/.puppetlabs/opt/facter/facts.d`).  I'll poke around and see if there's a missing step in the discourse_bench Dockerfile or something.

(As a later change—unless you really want it right now—I could add "test" and "bench" targets that do the above commands.  They could even be them dependencies of "all" so that they are run by default.)